packages/engine/Source/Scene/Scene.js

```diff
@@ -734,6 +734,12 @@ function Scene(options) {
   this.initializeFrame();
 }
 
+/**
+ * Use this to set the default value for {@link Scene#logarithmicDepthBuffer} in newly constructed Scenes
+ * This property relies on fragmentDepth being supported.
+ */
+Scene.defaultLogDepthBuffer = true;
```

Comment:
This addition should be listed in `CHANGES.md` since it's part of the public API.